### PR TITLE
ENT-2820 Document database tables

### PR DESCRIPTION
**Pull request by tudor-malene**
See: https://github.com/corda/enterprise/pull/1954/

Removed ent-only table and references

### CODE REVIEW

File: docs/source/node-database-tables.rst [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+             data could lead to unexpected behaviour and inconsistent views of the ledger.
+
+Network map
+===========
+
+
+Node Infos
+-----------
+
+
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: Delete extra blank line

File: docs/source/node-database-tables.rst [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+PARTICIPANTS                        X500 name of participant.
+==============================   ==========================================================================================
+
+
+
+
+Liquibase database migration
+============================
+
+These are `Liquibase <https://www.liquibase.org`_ proprietary tables used by Corda internally to manage schema change and evolution.
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: You're missing the `>` for the link

File: docs/source/node-database-tables.rst [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+==============================   ==========================================================================================
+PROPERTY_KEY                        The key
+PROPERTY_VALUE                      The value
+==============================   ==========================================================================================
+
+
+Vault tables
+============
+
+Read more about the vault here :doc:`vault.rst`.
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: Remove the `.rst`

File: docs/source/node-database-tables.rst [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+PUBLIC_KEY_HASH                     Public key hash
+==============================   ==========================================================================================
+
+These tables should be append only.
+
+
+Node state machine
+------------------
+
+Read more :doc:`node-services` DBCheckpointStorage
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: This line doesn't make sense. What is `DBCheckpointStorage` doing at the end?

File: docs/source/node-database-tables.rst [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+==============================   ==========================================================================================
+
+
+This table should be empty when no events are scheduled.
+
+
+Storage of private keys
+-----------------------
+
+Read more  :doc:`cryptoservice-configuration`
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: Read more in

File: docs/source/node-database-tables.rst [hunk 6]
```diff
@@ -0,0 +1,430 @@
...
+==============================   ==========================================================================================
+
+This table should be empty when no states are authorised for upgrade or after authorised states have been upgraded.
+
+
+
+Scheduling
+----------
+
+Read more :doc:`event-scheduling`
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: Read more in

File: docs/source/node-database-tables.rst [hunk 7]
```diff
@@ -0,0 +1,430 @@
...
+TRANSACTION_VALUE                 The binary representation of the transaction.
+STATE_MACHINE_RUN_ID              The flow id associated with this transaction.
+==============================   ==========================================================================================
+
+                                                                                     |
+
+Contract upgrades
+-----------------
+
+Read more :doc:`contract-upgrade`
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: Read more in

File: docs/source/node-database-tables.rst [hunk 8]
```diff
@@ -0,0 +1,430 @@
...
+The ledger data is formed of transactions and attachments.
+In future versions this data will be encrypted using SGX.
+Read more :doc:`key-concepts-ledger`
+
+Attachments
+-----------
+
+Read more: :doc:`tutorial-attachments` and :doc:`node-services`
+
+.. image:: resources/database/attachments_tables.png
```
Comment (shamsasari) [inline docs/source/node-database-tables.rst]: You haven't included these image files

File: docs/source/node-database-tables.rst (hunk 5 above)
Comment (tudor-malene) [reply docs/source/node-database-tables.rst]: This is an ent only doc. I have to remove it

File: docs/source/node-database-tables.rst (hunk 8 above)
Comment (tudor-malene) [reply docs/source/node-database-tables.rst]: oops

**Comment by tudor-malene** [comment]
back to you @shamsasari

File: docs/source/node-database-tables.rst (hunk 7 above)
Comment (shamsasari) [reply docs/source/node-database-tables.rst]: you missed this

File: docs/source/node-database-tables.rst (hunk 6 above)
Comment (shamsasari) [reply docs/source/node-database-tables.rst]: and this

**Pull request merged**